packages/n4s/src/rules/isValueOf.ts

```diff
@@ -0,0 +1,13 @@
+import bindNot from 'bindNot';
+import { isNotNullish } from 'isNullish';
+import { isString, isPlainObject } from 'lodash';
```

Comment:
Vest doesn't bring any third party dependencies, let's remove this and use whatever we have within Vest instead.
```suggestion
```

Reply:
removed external dependencies